--- a/src/plugins/messageservices/imap/imapservice.cpp
+++ b/src/plugins/messageservices/imap/imapservice.cpp
@@ -345,38 +345,7 @@ bool ImapService::Source::exportUpdates(const QMailAccountId &accountId)
         _service->errorOccurred(QMailServiceAction::Status::ErrInvalidData, tr("No account specified"));
         return false;
     }
-
-    //sync disconnected move and copy operations for account
-
-    QMailAccount account(accountId);
-
-    QMailFolderIdList folderList = QMailStore::instance()->queryFolders(QMailFolderKey::parentAccountId(accountId));
-
-    _service->_client.strategyContext()->moveMessagesStrategy.clearSelection();
-
-    bool pendingDisconnectedOperations = false;
-
-    foreach(const QMailFolderId& folderId, folderList)
-    {
-        if(!folderId.isValid())
-            continue;
-
-        QMailMessageKey movedIntoFolderKey = QMailMessageKey::parentFolderId(folderId)
-                                                 & (QMailMessageKey::previousParentFolderId(QMailFolderKey::parentAccountId(accountId))
-                                                    | QMailMessageKey::status(QMailMessage::LocalOnly));
-
-        QMailMessageIdList movedMessages = QMailStore::instance()->queryMessages(movedIntoFolderKey);
-
-        if(movedMessages.isEmpty())
-            continue;
-
-        pendingDisconnectedOperations = true;
-        _service->_client.strategyContext()->moveMessagesStrategy.appendMessageSet(movedMessages,folderId);
-    }
-
-    if(pendingDisconnectedOperations)
-        appendStrategy(&_service->_client.strategyContext()->moveMessagesStrategy, SIGNAL(messagesMoved(QMailMessageIdList)));
-
+    
     _service->_client.strategyContext()->exportUpdatesStrategy.clearSelection();
     appendStrategy(&_service->_client.strategyContext()->exportUpdatesStrategy);
     if(!_unavailable)
